enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/PresenceCheckService.java

```diff
@@ -335,4 +335,10 @@ private IdentityVerificationEntity fetchIdVerification(OwnerId ownerId) throws P
         return idVerification;
     }
 
+    private SessionInfo updateSessionInfo(final OwnerId ownerId, final IdentityVerificationEntity identityVerification, final SessionInfo sessionInfo) {
+        sessionInfo.getSessionAttributes().put("timestampLastUsed", ownerId.getTimestamp());
+        identityVerification.setSessionInfo(jsonSerializationService.serialize(sessionInfo));
+        return sessionInfo;
```

Comment:
It would be nice to better group the token and timestamp together under one field, e.g. if we wanted to add a `timestampCreated`. I also would suggest to unify the camel case, so that the JSON matches other JSONs we use.

Something like:

```json
{
  "sessionAttributes": {
    "iProovVerificationToken": {
       "tokenId": "8e489404-74ad-4412-a2cb-f71e329c6183",
       "timestampLastUsed": 1667906646007
     }
  }
}
```

And for mock:
```json
{
  "sessionAttributes": {
    "mockVerificationToken": {
       "tokenId": "8e489404-74ad-4412-a2cb-f71e329c6183",
       "timestampLastUsed": 1667906646007
     }
  }
}
```

This is just a suggestion, feel free to ignore.

Reply:
This sounds good. I unified camelCase but let's postpone grouping of attributes to another PR. The current map contains objects, so I see possible NPE...

Reply:
Ok, sure.